src/main/java/org/candlepin/subscriptions/db/model/Granularity.java

```diff
@@ -20,17 +20,55 @@
  */
 package org.candlepin.subscriptions.db.model;
 
+import org.candlepin.subscriptions.utilization.api.model.GranularityGenerated;
+
 /**
  * Granularity of a given snapshot.
  *
  * Granularity defines the scope of max concurrent usage. For example, max concurrent usage across a week
  * represents the maximum tally totals across all days in that week. For example, given a week where daily
  * tallies were 2, 3, 4, 5, 6, 2, 4, the weekly tally snapshot would be 6.
  */
-public enum Granularity {
-    DAILY,
-    WEEKLY,
-    MONTHLY,
-    QUARTERLY,
-    YEARLY
+public enum Granularity implements EnumLookupUtil {
+    DAILY("Daily") {
+        @Override
+        public GranularityGenerated asOpenApiEnum() {
```

Comment:
This is producing this warning...
![image](https://user-images.githubusercontent.com/19955562/101080812-416d6200-3577-11eb-8834-7f8795c30c79.png)

not entiiiiiirely sure that I'm concerned about it yet?  Open to suggestions.

Note: the asOpenApiEnum returns `<T extends Enum<T>> T`

Reply:
After latest refactor, this isn't relevant anymore